### DISCUSSION TITLE
Can this operator find and label nodes with Intel quicksync?

### DISCUSSION THREAD

**Discussion by 2fst4u** [kubernetes-sigs/node-feature-discovery#1203] [General]
I've tried scouring the docs and I can't find any reference to it but I'm not too technical so I'm not sure if I'm missing it under a different name maybe. Can quicksync nodes be labelled?

**Comment by marquiz** [comment]
NFD does not have any built-in "quick sync" label. TBH, I'm not familiar with the technology but I suspect that quick sync acceleartor  is available as a separate PCI device in the system(?) If so, you should be able write a [NodeFeatureRule](https://kubernetes-sigs.github.io/node-feature-discovery/v0.13/usage/customization-guide.html#nodefeaturerule-custom-resource). But then the question becomes, what do you want to do with the label, e.g. how do you make the HW actually available for workload(s)?
